wazo_acceptance/steps/user.py

```diff
@@ -92,7 +92,15 @@ def given_there_are_telephony_users_with_infos(context):
             device = context.confd_client.devices.list(mac=body['device'])['items'][0]
             context.helpers.line.add_device(line, device)
 
-        # TODO voicemail
+        if (body.get('voicemail_name')
+                and body.get('voicemail_number')
+                and body.get('voicemail_context')):
+            voicemail = context.helpers.voicemail.create({
+                'name': body['voicemail_name'],
+                'context': body['voicemail_context'],
+                'number': body['voicemail_number']
+            })
+            context.helpers.voicemail.associate(confd_user, voicemail['id'])
```

Comment:
To avoid another name layer with the confd-client: 
```suggestion
            context.helpers.voicemail.add_user(voicemail['id'], confd_user)
```
or 
```suggestion
            context.helpers.confd_user.add_voicemail(confd_user, voicemail['id'])
```

Reply:
I took the second one, to be symmetric with confd_user.add_line()